train/trainer.py

```diff
@@ -200,11 +200,14 @@ def training_step(self, batch, batch_idx):
         self.log("train_batch_loss", losses["main"].item())
         # for the lr scheduler
 
+        self.weight_norms = self.get_weight_norms()
```

Comment:
why are you storing these instead of just running `self.log_stat("weight_norms", self.get_weight_norms())` below? are you using them somewhere?